### ISSUE TITLE
Incorrectly build netplan yaml 

### ISSUE THREAD

**Issue by albosmart-ro** [dallgoot/yaml#7]
Expected yaml 

```
network:
  ethernets:
        enp0s3:
            addresses: ['192.168.1.84/24']
            gateway4: 192.168.1.1
            nameservers:
                addresses: [192.168.1.1]
  version: 2

```
got
```
network:
  ethernets:
    enp0s3:
      addresses: [192.168.1.84\/24]
      gateway4: 192.168.1.1
      nameservers:
addresses: [192.168.1.1]
  version: 2
```
Code

```
$yaml = new Dallgoot\Yaml\YamlObject(0);

    $network = new \StdClass;
        $ethernets = new \StdClass;

            $enp0s3 = new \StdClass;
            $string = '192.168.1.84/24';
            $enp0s3->addresses = new Dallgoot\Yaml\Compact(array($string));
            $enp0s3->gateway4 = '192.168.1.1';
                $nameservers =  new \StdClass;
                $nameservers->addresses = new Dallgoot\Yaml\Compact(['192.168.1.1']);
            $enp0s3->nameservers = $nameservers;
        $ethernets->enp0s3 = $enp0s3;
    $network->ethernets = $ethernets;
    $network->version = 2;
$yaml->network = $network;
$text = Dallgoot\Yaml::dump($yaml, 0);
file_put_contents('00-installer-config.yaml',$text);
```

**Comment by dallgoot** [comment]
thank you for the feedback and the code example and sorry for the delay for some reason i didn't have the notification

it seems that it loses the correct indentation for the last element of **enp0s3**
As a temporary workaround try adding another last empty property to your **enp0s3** object after **nameservers** :
`$enp0s3->ignoreme = ' '`
and tell me if helps

i'll come back to you ASAP for a fix

**Comment by dallgoot** [comment]
Fixed in 0.9.0.1
